### ISSUE TITLE
Add ability to create a new record

### ISSUE THREAD

**Issue by nick-herrera** [EditLLC/montage-frontend#43]

**Comment by juliancoleman** [comment]
I'm going to design the routes and components, however, I'm going to wait on adding the button to `create` until after @Karvel finishes the tabs on the query builder. Then I'll see if I can't just add a button to the tabs element as perhaps a toolbar item and push it over to the right side of the page.

**Comment by nick-herrera** [comment]
Duplicate of #26
